spec/requests/lobbies_spec.rb

```diff
@@ -327,4 +335,56 @@
       end
     end
   end
+
+  describe 'players_done' do
+    let(:lobby) { FactoryBot.create(:lobby) }
+    let(:question) { FactoryBot.create(:question, quiz: lobby.quiz) }
+    let(:playerA) { FactoryBot.create(:player, lobby: lobby) }
+    let(:playerB) { FactoryBot.create(:player, lobby: lobby) }
+
+
+    before do
+      answer = FactoryBot.create(:answer, question: question)
+
+      playerA
+      playerB
+
+      FactoryBot.create(:player_answer, player: playerA, answer: answer)
+    end
+
+    subject { get players_done_api_v1_lobby_path(id: lobby.id), params: { question_id: question.id } }
+
+    it 'responds with successful HTTP status' do
+      subject
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'responds with the players\'s that have answered the question' do
+      subject
+
+      parsed_response = JSON.parse(response.body)
+      expect(parsed_response).to eq([playerA].as_json)
+    end
+  end
 end
+
+describe 'score' do
+  let(:lobby) { FactoryBot.create(:lobby) }
+  let(:playerA) { FactoryBot.create(:player, lobby: lobby) }
+  let(:playerB) { FactoryBot.create(:player, lobby: lobby) }
+
+  before do
+    allow(Pusher).to receive(:trigger)
+    playerA
+    playerB
+  end
+
+  subject { get score_api_v1_lobby_path(id: lobby.id) }
+
+  it 'responds with successful HTTP status' do
+    subject
+
+    expect(response).to have_http_status(:success)
+  end
```

Comment:
You should also test that the scores are returned correctly. Create all necessary models so that you can test the response as well!